index.html

```diff
@@ -1,50 +1,274 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=, initial-scale=1.0">
-    <title>MyPortfolio</title>
-    <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-    <section class="container">
-        <header class="upperHeader">
-            <nav class="upperNavbar">
-                <a class="link" href="">Hamdullah</a>
-                <img class="flexend" src="images/Union.png" alt="">
-            </nav>
-        </header>
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=, initial-scale=1.0">
+        <title>MyPortfolio</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
```

Comment:
- [ ] I see `background-color: aqua` on the body element which is not required for this project as it makes your project differ from the Figma design. Please remove this. 
- [ ] Also, you have default `margin: 8px;` on the body element and it is also affecting your design negatively as well. Please fix this.